pkg/config/config_test.go

```diff
@@ -14,50 +13,86 @@ func Test_ReadConfig(t *testing.T) {
 		err    string
 	}{
 		"serviceIDs required": {
-			env:    map[string]string{},
+			env: map[string]string{
+				EnvUsername:  "user",
+				EnvPassword:  "password",
+				EnvNamespace: "namespace",
+			},
 			config: nil,
-			err:    "OSB_SERVICE_IDS is required",
+			err:    "OSB_SERVICE_IDS is required, but was not defined or is empty",
+		},
+		"empty serviceIDs given": {
+			env: map[string]string{
+				EnvUsername:   "user",
+				EnvPassword:   "password",
+				EnvNamespace:  "namespace",
+				EnvServiceIDs: ",,,",
+			},
+			config: nil,
+			err:    "OSB_SERVICE_IDS is required, but was not defined or is empty",
 		},
 		"username required": {
 			env: map[string]string{
-				"OSB_SERVICE_IDS": "1,2,3",
+				EnvServiceIDs: "1,2,3",
 			},
 			config: nil,
-			err:    "OSB_USERNAME is required",
+			err:    "OSB_USERNAME is required, but was not defined or is empty",
 		},
 		"password required": {
 			env: map[string]string{
-				"OSB_SERVICE_IDS": "1,2,3",
-				"OSB_USERNAME":    "user",
+				EnvServiceIDs: "1,2,3",
+				EnvUsername:   "user",
 			},
 			config: nil,
-			err:    "OSB_PASSWORD is required",
+			err:    "OSB_PASSWORD is required, but was not defined or is empty",
```

Comment:
Maybe use the variable here as well?
```suggestion
			err:    EnvPassword + " is required, but was not defined or is empty",
```